Dockerfile

```diff
@@ -39,4 +39,10 @@ RUN dnf update -y && dnf install -y --setopt=tsflags=nodocs \
       hadolint \
       && dnf clean all
```

Comment:
please match the dependencies to the list here: https://github.com/redhat-aqe/tox-container/blob/python-3.6/Dockerfile#L18

Reply:
Thank you for pointing this out. Fixed now.